python/lsst/ctrl/mpexec/cli/opt/options.py

```diff
@@ -290,3 +290,14 @@ def _to_int(value):
                                      help=unwrap("""Stop processing at first error, default is to process
                                                  as many tasks as possible."""),
                                      is_flag=True)
+
+save_execution_butler_option = MWOptionDecorator("--save-execution-butler",
+                                                 help=unwrap("""Export location for an
+                                                             execution-specific butler after making
+                                                             QuantumGraph"""),
+                                                 type=MWPath(dir_okay=True, file_okay=False, writable=True))
```

Comment:
We don't want to tell click it's a local file path if we intend to allow people to write these to object stores directly. Since we can support that option we probably shouldn't force the command line to disallow it.